ui/src/dataExplorer/components/DataExplorerPage.tsx

```diff
@@ -25,7 +25,7 @@ const DataExplorerPage: FC = ({children}) => {
   return (
     <Page titleTag={pageTitleSuffixer(['Data Explorer'])}>
       {children}
-      <GetResources resources={[ResourceType.Variables, ResourceType.Buckets]}>
+      <GetResources resources={[ResourceType.Variables]}>
```

Comment:
I'm worried that `setBuilderBucketIfExists` function runs in `DataExplorer.tsx` before `loadBuckets` is called, so will not set buckets from query parameters, if buckets are not available yet. There are a couple ways we can fix this.. let's talk about it.

Reply:
taking this back.